1-WebApp-OIDC/1-1-MyOrg/Startup.cs

```diff
@@ -32,16 +34,19 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             // Sign-in users with the Microsoft identity platform
-            services.AddMicrosoftIdentityPlatformAuthentication(Configuration);
+            //services.AddSignIn(Configuration);
+            services.AddAuthentication(OpenIdConnectDefaults.AuthenticationScheme)
+                .AddSignIn("AzureAD", Configuration, options => Configuration.Bind("AzureAD", options));
 
             services.AddControllersWithViews(options =>
             {
                 var policy = new AuthorizationPolicyBuilder()
                     .RequireAuthenticatedUser()
                     .Build();
                 options.Filters.Add(new AuthorizeFilter(policy));
-            });
-           services.AddRazorPages();
+            }).AddMicrosoftIdentityUI();
```

Comment:
By removing `AzureAD.UI` packages, all of our samples needed an action result for sign-in, sign-out (and edit profile for B2C). A razor project called Microsoft.Identity.UI was created with a standard action result and views for these cases, however it is completely optional to use them.